WebApplication/Backend/Services/FeedbackService.cs

```diff
@@ -15,35 +20,78 @@ public FeedbackService()
 
         }
 
-        internal List<Feedback> GetAllFeedbacks()
+        internal Feedback GetFeedbackById(string id)
         {
             throw new NotImplementedException();
         }
 
-
-        internal Feedback GetFeedbackById(string id)
+        ///Tanja Drcelic RA124/2017
+        /// <summary>
+        ///calls method for get all feedback in feedback table
+        ///</summary>
+        ///<returns>
+        ///list of feedbacks
+        ///</returns>
+        internal List<Feedback> GetAllFeedbacks()
         {
-            throw new NotImplementedException();
+            return feedbackRepository.GetAllFeedbacks();
         }
 
+        ///Aleksandra Milijevic RA 22/2017
+        /// <summary>
+        ///calls method for get approved feedbacks from feedback table
+        ///</summary>
+        ///<returns>
+        ///list of approved feedbacks
+        ///</returns>
+
         internal List<Feedback> GetApprovedFeedbacks()
         {
-            throw new NotImplementedException();
+            return feedbackRepository.GetFeedbacks(true);
         }
 
-        public void DeleteFeedback(Feedback feedback)
+        ///Tanja Drcelic RA124/2017
+        /// <summary>
+        ///calls method for get disapproved feedback from feedback table
+        ///</summary>
+        ///<returns>
+        ///list of not approved feedbacks
+        ///</returns>
+        internal List<Feedback> GetNotApprovedFeedbacks()
         {
-            throw new NotImplementedException();
+
+            return feedbackRepository.GetFeedbacks(false);
         }
 
-        public void EditFeedback(Feedback feedback)
+        public void DeleteFeedback(Feedback feedback)
         {
             throw new NotImplementedException();
         }
-
-        public void NewFeedback(Feedback feedback)
+        ///Marija Vucetic 
+        /// <summary>
+        ///calls method for set na value of attribute Approved
+        ///</summary>
+        ///<returns>
+        ///list of not approved feedbacks
+        ///</returns>
+        ///<param name="feedback"> Feedback type object
+        ///</param>>
+        public void EditFeedback(FeedbackDTO feedback)
         {
-            throw new NotImplementedException();
+             feedbackRepository.SetFeedbackApprovedValue(feedback);
+        }
```

Comment:
This method should be called ApproveFeedback as that is the only field being changed